sdk/formrecognizer/azure-ai-formrecognizer/tests/test_compose_model_async.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import pytest
+import functools
+from azure.core.exceptions import ClientAuthenticationError, HttpResponseError
+from azure.ai.formrecognizer.aio import FormTrainingClient
+from testcase import FormRecognizerTest, GlobalFormRecognizerAccountPreparer
+from testcase import GlobalClientPreparer as _GlobalClientPreparer
+from asynctestcase import AsyncFormRecognizerTest
+
+
+GlobalClientPreparer = functools.partial(_GlobalClientPreparer, FormTrainingClient)
+
+
+class TestTrainingAsync(AsyncFormRecognizerTest):
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer(training=True)
+    async def test_compose_model(self, client, container_sas_url):
+        async with client:
+            poller = await client.begin_training(container_sas_url, use_training_labels=True)
+            model_1 = await poller.result()
+
+            poller = await client.begin_training(container_sas_url, use_training_labels=True, display_name="second-labeled-model")
+            model_2 = await poller.result()
+
+            poller = await client.begin_create_composed_model([model_1.model_id, model_2.model_id], display_name="my composed model")
+
+            composed_model = await poller.result()
+            self.assertEqual(composed_model.display_name, "my composed model")
+            self.assertComposedModelHasValues(composed_model, model_1, model_2)
+
```

Comment:
can you add tests for compose model without display_name and calling compose model with api version 2.0